tokio/Cargo.toml

```diff
@@ -54,7 +54,15 @@ io-util = ["memchr"]
 # stdin, stdout, stderr
 io-std = ["rt-core"]
 macros = ["tokio-macros"]
-net = ["dns", "tcp", "udp", "uds"]
+net = [
+  "dns",
+  "lazy_static",
+  "libc",
+  "mio/os-poll",
+  "mio/tcp",
+  "mio/udp",
+  "mio/uds",
```

Comment:
I feel like if I want tcp I really don't want to compile `mio-uds`?

Reply:
It doesn't add [any new dependencies](https://github.com/tokio-rs/mio/blob/a5381b3e477ece2ef0db57adafad145746bd33cb/Cargo.toml#L29-L35). (also, we already [depends on `libc` via `mio`.](https://github.com/tokio-rs/mio/blob/a5381b3e477ece2ef0db57adafad145746bd33cb/Cargo.toml#L41)) I don't think they actually have a visible effect on compile time.

Reply:
That is fair, ill leave this up to @carllerche

Reply:
I think it is fine given the size of `uds`. we can always split it up later if we need to again. We have no metrics on compile-time & bin size w/ various feature flags, so I would opt for reducing here.